### PR TITLE
Add Secrets Provider helm chart version 1.1.0

### PR DESCRIPTION
**Pull request by abrahamko**
Secrets Provider supports helm chart, so it is added to here too.

### CODE REVIEW

File: docs/index.yaml
Comment (sigalsax) [inline docs/index.yaml:5]: @izgeri @abrahamko are these intentional changes or something that gets changed automatically?

**Comment by sigalsax** [comment]
@abrahamko LGTM! Let's wait until we push a release before merge. Thanks!

File: docs/index.yaml
Comment (abrahamko) [reply docs/index.yaml:5]: All the changes in the index file are the result of running the `reindex.sh` script.

**Review by izgeri — APPROVED** [review]
LGTM

**Review by sigalsax — APPROVED** [review]
LGTM!

**Pull request merged**
